src/main/java/com/force/i18n/grammar/LanguageDictionary.java

```diff
@@ -92,6 +96,13 @@ public LanguageDictionary(HumanLanguage language, LanguageDictionary from) {
         this.isSkinny = from.isSkinny;
     }
 
+    protected void createGrammaticalTermMap() {
+        this.nounMap = new GrammaticalTermMapImpl<Noun>();
```

Comment:
redundant.  JDK 11 does not require type 
```suggestion
        this.nounMap = new GrammaticalTermMapImpl<>();
```